recipe/meta.yaml

```diff
@@ -45,8 +45,8 @@ requirements:
 
   run:
     - python
-    - libgfortran  # [not win]
-    - libgfortran5  # [not win]
```

Comment:
shouldn't these "c" library requirements be automatically exported from the fortran compilers in the build section? i feel like you don't need these at all.